src/pages/leejiyon/Main/Components/Article.js

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from 'react';
+import Comments from './Comments';
+import './Article.scss';
+
+const emptyHeartStyle = {
+  backgroundPosition: `-156px -478px`,
+};
+const likedHeartStyle = {
+  backgroundPosition: `-130px -478px`,
+};
+
+class Article extends Component {
+  inputRef = React.createRef();
+  state = {
+    isArticleLiked: false,
+    isCommentInputEntered: false,
+    commentInputValue: '',
+    currentUser: 'hao',
+    lastPostedCommentTIme: '42분 전',
+    commentLikeNum: '9',
+    userComments: [
+      {
+        id: 1,
+        userId: 'iu',
+        comment: '아자아자',
+        showTrashBtn: false,
+        liked: true,
+      },
+      {
+        id: 2,
+        userId: 'jiyon',
+        comment: '공부를 하자',
+        showTrashBtn: false,
+        liked: false,
+      },
+      {
+        id: 3,
+        userId: 'Shaman_king',
+        comment: '벨로그 쓰기',
+        showTrashBtn: false,
+        liked: false,
+      },
+    ],
+    id: 4,
+  };
+
+  onClickArticleHeartBtn = () => {
+    this.setState({
+      isArticleLiked: !this.state.isArticleLiked,
+    });
+    if (this.state.commentLikeNum === 10) {
+      this.setState({
+        commentLikeNum: 9,
+      });
+    } else {
+      this.setState({
+        commentLikeNum: 10,
+      });
+    }
+  };
+
+  onClickPostCommentInput = e => {
+    if (e.keyCode === 13) {
+      e.preventDefault();
+      this.onClickPostBtn();
+    }
+  };
+
+  onChangeCommentInput = e => {
+    this.setState({
+      commentInputValue: e.target.value,
+    });
+    if (e.target.value.length > 0) {
+      this.setState({
+        isCommentInputEntered: true,
+      });
+    } else {
+      this.setState({
+        isCommentInputEntered: false,
+      });
+    }
+  };
+
+  onClickPostBtn = () => {
+    if (!this.state.commentInputValue.length) {
+      return;
+    }
+    this.setState({
+      userComments: this.state.userComments.concat({
+        id: this.state.id,
+        userId: this.state.currentUser,
+        comment: this.state.commentInputValue,
+        showTrashBtn: true,
+      }),
```

Comment:
```suggestion
      userComments: [...this.state.userComments,{
        id: this.state.id,
        userId: this.state.currentUser,
        comment: this.state.commentInputValue,
        showTrashBtn: true,
      }]
```

배열의 병합 같아보이는데 spread operator 사용하시면 더욱 좋아질 것 같습니다!

Reply:
확인했습니다